aspnetcore/blazor/class-libraries.md

```diff
@@ -126,3 +126,28 @@ dotnet nuget publish
 ```
 
 When using the `blazorlib` template, static resources are included in the NuGet package. Library consumers automatically receive scripts and stylesheets, so consumers aren't required to manually install the resources. Note that [static assets aren't supported for server-side apps](#static-assets-not-supported-for-server-side-apps), including when a Blazor library (`blazorlib`) is referenced by a server-side app.
+
+## Create a razor class library with static assets
+
+Razor Class Libraries (RCL) frequently require companion static assets that can be to be referenced by the consuming app of the RCL. ASP.NET Core allows creating RCLs that include static assets that are available from a consuming app.
```

Comment:
```suggestion
Razor class libraries (RCL) frequently require companion static assets that can be referenced by the consuming app of the RCL. ASP.NET Core allows creating RCLs that include static assets that are available to a consuming app.
```

Thus far, we've been going without the proper noun for RCL. If you want to change that, sounds good to me, but it should be updated everywhere in the topic (and perhaps even elsewhere across the docs). If you want, I can take that up later on another PR (or perhaps I'll just update it as I go on the Preview 6 updates coming up shortly).